examples/transaction.py

```diff
@@ -1,10 +1,10 @@
-import berbix
 import sys
 import os
 import pprint
 import time
 
 sys.path.insert(1, os.path.join(sys.path[0], '..'))
+import berbix
```

Comment:
nit: I'm guessing it's normal python imports formatting to import this here instead of with the other packages?

Reply:
this is for local debugging so it picks up the local path instead of the pip package

Reply:
ah ok, thanks for clarifying